lua/nvim-navic/init.lua

```diff
@@ -16,8 +16,10 @@ local function request_symbol(for_buf, handler, client_id)
 					request_symbol(for_buf, handler, client_id)
 				end, 750)
 			elseif symbols[client_id].result ~= nil then
-				vim.b[for_buf].navic_awaiting_lsp_response = false
-				handler(for_buf, symbols[client_id].result)
+				if vim.api.nvim_buf_is_valid(for_buf) then
```

Comment:
would nvim_buf_is_loaded be more appropriate here?

Reply:
I don't understand you

Reply:
Its fine, I will add it my self. Thanks for the PR! 😄